assets/js/modules/adsense/settings/adsense-settings.js

```diff
@@ -135,9 +139,34 @@ class AdSenseSettings extends Component {
 		toggleConfirmModuleSettings( 'adsense', settingsMapping, this.state );
 	}
 
+	generateAccountStatusLabel() {
+		const { accountStatus } = this.state;
+		switch ( accountStatus ) {
+			case 'account-connected':
+				return __( 'Your account has been approved.', 'google-site-kit' );
+
+			case 'account-pending-review':
+				return __( 'We\'re getting your site ready for ads. This usually takes less than a day, but it can sometimes take a bit longer.', 'google-site-kit' );
```

Comment:
```suggestion
				return __( 'We’re getting your site ready for ads. This usually takes less than a day, but it can sometimes take a bit longer.', 'google-site-kit' );
```